fluent-bit-to-loki/pkg/lokiplugin/loki.go

```diff
@@ -70,6 +70,17 @@ func (l *loki) SendRecord(r map[interface{}]interface{}, ts time.Time) error {
 	records := toStringMap(r)
 	level.Debug(l.logger).Log("msg", "processing records", "records", fmt.Sprintf("%+v", records))
 	lbs := model.LabelSet{}
+
+	if l.cfg.ExtractKubernetesMetadataFromTag {
+		if _, ok := records["kubernetes"]; !ok {
```

Comment:
Why is this needed?
The metadata extraction from the `tag` is added as a fallback in case of failure of the metadata or it completely replaces it?